serverjs/carddb.js

```diff
@@ -160,11 +160,11 @@ function getIdsFromName(name) {
     const split = name.split('[');
     return getIdsFromName(split[0])
       .map((id) => cardFromId(id))
-      .filter((card) => normalizeName(card.full_name) === name)
+      .filter((card) => getNameForComparison(card.full_name) === getNameForComparison(name))
```

Comment:
This is necessary because the frontend will now try to post special character names, which it didn't previously.